symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/SymphonyBdkAppProperties.java

```diff
@@ -0,0 +1,18 @@
+package com.symphony.bdk.app.spring;
+
+import com.symphony.bdk.app.spring.properties.AppAuthProperties;
+import com.symphony.bdk.core.config.model.BdkExtAppConfig;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration Properties for the Symphony BDK Extension App.
+ */
+@Getter
+@Setter
+@ConfigurationProperties(prefix = "bdk.app")
+public class SymphonyBdkAppProperties extends BdkExtAppConfig {
+  private AppAuthProperties auth;
+}
```

Comment:
Not sure that this is the right way to manage starters properties. IMHO we should have 2 separate entries in `application.yaml` file: 
```yaml
bdk:     # core starter config
   host: acme.symphony.com
bdk-app: # ext app starter config
   auth: 
     enabled: true
     jwtCookie: 
        enabled: true
        expireIn: 43200
```